src/lando/main/tests/__init__.py

```diff
@@ -66,3 +66,21 @@ def test_require_phabricator_api_key(monkeypatch, optional, valid_key, status):
         assert resp.body.api_token == "custom-key"
 
     assert resp.status_code == status
+
+
+@pytest.mark.django_db(transaction=True)
+def test_phabricator_api_key_encryption():
+    user = User.objects.create_user(username="test_user", password="test_password")
+    profile = Profile.objects.create(user=user)
+
+    assert profile.phabricator_api_key == ""
+
+    # Set an arbitrary key.
+    key = "test-key"
+    profile.save_phabricator_api_key(key)
+    assert key.encode("utf-8") not in profile.encrypted_phabricator_api_key
+    assert profile.phabricator_api_key == key
+
+    # Clear the key
+    profile.clear_phabricator_api_key()
+    assert profile.phabricator_api_key == ""
```

Comment:
Nit: the comments in this test would be better as assert strings, so they show up as error messages in test failure output. Something like:

```suggestion
    profile.clear_phabricator_api_key()
    assert profile.phabricator_api_key == "", "`clear_phabricator_api_key` should result in an empty string in the API key on the profile."
```